README.md

```diff
@@ -204,8 +206,10 @@ You may encounter an issue with the `unused_deps` check failing on the CI while
 
 ## Changelog
 
-See [CHANGELOG.md](https://github.com/karolsluszniak/ex_check/blob/master/CHANGELOG.md).
+See [CHANGELOG.md](./CHANGELOG.md).
```

Comment:
I think this will break the link when README is presented on HexDocs. Therefore I think we need to revert this.

Reply:
It's included as extras documents to `ex_doc`, the relative link would work just fine.

```elixir
 46   defp docs do
 47     [
 48       extras: [
 49         "CHANGELOG.md": [title: "Changelog"],
 50         "LICENSE.md": [title: "License"],
 51         "README.md": [title: "Overview"]
 52       ],
```

Reply:
It won't because `README.md` results in routing to `readme.html` so e.g. `CHANGELOG.md` will be at `changelog.html` while your link leads to `./CHANGELOG.md` which returns 404.

Reply:
@karolsluszniak I've tested it locally, the relative links seemed working? Did I miss out something?

![20220215_210450](https://user-images.githubusercontent.com/134518/154068800-f14601a5-7dd0-41c0-812f-7582fe254b69.png)

![20220215_210609](https://user-images.githubusercontent.com/134518/154068833-f7106793-0639-426b-88ab-591dad0b008e.png)

Reply:
:+1: It'd seem that ex_check does some some link substitution based on what's in `extras`. OK then - my mistake and thanks for checking - should've done it myself in the first place.